index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+  <style>
+      *{
+    margin-left: 30px;
+    padding: 0;
+}
+
+body{
+    background-color:#eee ;
+
+}
+ul{
+  text-decoration: none;
+}
+
+#nav-bar{
+    top: 0px;
+    display: flex;
+    width: 100%;
+}
+
+@media print {
+    body {
+        font-size: 12pt;
+    }
+}
+
+
+@media only screen and (max-width: 80px){
+    body{
+        background-color: gray;
+    }
+
+}
+
+
+@media only screen and (max-width: 800px){
+    #nav-bar{
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+  .product{
+    border: solid #f000f0 3px;
+    border-radius: 2px;
+    text-align: center;
+  }
+    
+}
+    </style>
+  </head>
+
+  <body>
+
+    <header id="header">
```

Comment:
please use only external CSS styling. it helps keep the codebase clean and organized.

Reply:
Good review, I would suggest you use bullets to separate your points, it makes it more readable. eg


- please use only external CSS styling. it helps keep the codebase clean and organized